pybikes/socialbicycles.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+import json
+
+from pybikes.contrib import TSTCache
+from pybikes.utils import PyBikesScraper
+
+from .base import BikeShareStation, BikeShareSystem
+
+__all__ = ['SocialBicycles', 'SocialBicyclesStation']
+
+BASE_URL = 'https://app.socialbicycles.com/api/networks/{uid}/hubs?page={page}&per_page='
+
+# Since most networks share the same hostname, there's no need to keep hitting
+# the endpoint on the same urls. This caches the feed for 60s
+cache = TSTCache(delta=60)
```

Comment:
This system does not need a Timestamped Test Cache (TSTCache). This cache system was just a hack / stupid idea to support systems where one url gives multiple feeds at once.

Reply:
Ah, my bad. I got it copied from other system implementation.